pontoon/pretranslation/transformer.py

```diff
@@ -96,8 +96,8 @@ def create_locale_plural_variants(node: FTL.SelectExpression, locale: Locale):
 class PreparePretranslation(Transformer):
     """
     Flattens the given Pattern, uplifting selectors to the highest possible level and
-    duplicating shared parts in the variants. All other Placeables are serialised as
-    TextElements.
+    duplicating shared parts in the variants. Transforms plural variants to match the
+    locale. All other Placeables are serialised as TextElements.
```

Comment:
```suggestion
    locale.
```

Fixing my own mistake here. Nothing happens to the placeables in this transform.